.github/CODEOWNERS

```diff
@@ -1,6 +1,7 @@
 # Default owner for all directories not owned by others
 *                       @googleapis/yoshi-go-admins
 
+/ai/                    @jba @eliben @googleapis/yoshi-go-admins
```

Comment:
Instead of your names can you use the team you are in `@googleapis/go-vertexai`

Reply:
Done.